desktop.css

```diff
@@ -341,6 +340,7 @@ h5 {
 
 .about-btn {
   width: auto;
+  margin-bottom: 100px;
 }
 
```

Comment:
- [ ] BRILLIANT Job 💎 🙌 but there is one thing that you forgot to build. That would be good if you boild language cards on the right in about me section by yourself, instead of just putting screenshot of them. 

You can the img where you made a mistake
![image](https://user-images.githubusercontent.com/69011963/144710697-4133c3f3-9434-41c6-8ac2-f1b92c6dd769.png)